### DISCUSSION TITLE
Missing link to resource files on Netbeans webpage

### DISCUSSION THREAD

**Discussion by coshlo** [apache/netbeans#3581] [General]
HI i'm new to Netbeans and was trying out your [Getting Started with HTML5 Applications](https://netbeans.apache.org/kb/docs/webclient/html5-gettingstarted.html) tutorial but found the link to the resources file zip to be broken.

The link is under the 'Editing the HTML File' section named 'project resources' and links to "https://netbeans.org/projects/samples/downloads/download/Samples/Web%20Client/HTML5Demo-projectresources.zip"

Any chance this can get fixed?

**Comment by geertjanw** [comment]
Yes, you'll need to do without those resources, or create them via the tutorial and contribute them.
